graphbolt/src/fused_csc_sampling_graph.cc

```diff
@@ -1130,11 +1148,12 @@ static torch::Tensor NonUniformPickOp(
 template <typename PickedType>
 inline int64_t NonUniformPick(
     int64_t offset, int64_t num_neighbors, int64_t fanout, bool replace,
-    const torch::TensorOptions& options,
-    const torch::optional<torch::Tensor>& probs_or_mask,
+    const torch::TensorOptions& options, const torch::Tensor& probs_or_mask,
```

Comment:
@czkkkkkk Since NonUniformPick requires a probs_or_mask argument, I changed its type from an optional to a plain tensor.